lib/join_cluster.js

```diff
@@ -338,7 +338,8 @@ JoinCluster.prototype.joinNode = function joinNode(node, callback) {
     var self = this;
     var joinOpts = {
         host: node,
-        timeout: this.joinTimeout
+        timeout: this.joinTimeout,
+        service: 'ringpop'
```

Comment:
kewl. i'll eventually want to centralize this logic so that a call site doesnt "forget"